packages/server/lib/modes/run.coffee

```diff
@@ -677,10 +680,28 @@ module.exports = {
       ## is the one that matches our id!
       project.on("socket:connected", fn)
 
+  _delayToLetVideoFinish: ->
+    # we are recording video of the test run
+      # and there still might be frames sent by the browser
+      # in order to avoid chopping off the end of the video
+      # delay closing the browser by N ms
+      DELAY_TO_LET_VIDEO_FINISH_MS = 1000
```

Comment:
you could instead export + stub this delay value in the test to get extra coverage of this function